scripts/release-api.sh

```diff
@@ -23,7 +23,7 @@ fi
 echo "Setting API version to ${RELEASE_API_VERSION}"
 mvn -q versions:set -DnewVersion="${RELEASE_API_VERSION}" -DprocessAllModules=true -pl :kroxylicious-bom,${API_MODULES} -DgenerateBackupPoms=false
 mvn -q clean install -Pquick -pl :kroxylicious-bom,${API_MODULES}  #quick sanity check to ensure the API modules still build
```

Comment:
Not actually part of your PR but maybe this should be `mvn clean verify` instead of `install`

Reply:
I think you are right. I can fix this as part #672